src/examples/federation/src/backend/index.ts

```diff
@@ -0,0 +1,48 @@
+import Graphweaver from '@exogee/graphweaver-server';
+import { Server } from '@hapi/hapi';
+import hapiApollo from '@as-integrations/hapi';
+import { DirectiveLocation, graphweaverMetadata } from '@exogee/graphweaver';
+
+import './schema';
+
+// Register Custom Directive
+graphweaverMetadata.collectDirectiveTypeInformation({
+	name: 'custom',
+	locations: [DirectiveLocation.OBJECT],
+});
+
+export const graphweaver = new Graphweaver({
+	enableFederation: true,
+	enableFederationTracing: true,
+	schemaDirectives: {
+		composeDirective: {
+			name: '@custom',
+		},
+		link: {
+			url: 'https://myspecs.dev/myCustomDirective/v1.0',
+			import: ['@custom'],
+		},
+	},
+});
+
+const start = async () => {
+	await graphweaver.server.start();
+	// create the hapi server
+	const hapi = new Server({
+		host: '0.0.0.0',
+		port: 4001,
+	});
+
+	await hapi.register({
+		plugin: hapiApollo,
+		options: {
+			apolloServer: graphweaver.server,
+			path: '/',
+		},
+	});
+
+	await hapi.start();
```

Comment:
Why do we need both the Graphweaver server and Hapi here? Could we start it with `graphweaver start` using `sls offline` like we normally do?

Reply:
not easily as I would need to bring in all the workspace into the docker.

This seemed like the easiest solution, what do you think?

Reply:
Yup, sold.